### PR TITLE
Add example to demonstrate usage

### PR DESCRIPTION
**Pull request by Khazuar**

### CODE REVIEW

File: examples/example1/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+# Example 1
+
+This simple react-app uses both a `MathFieldComponent` and standart html input-control to display the same content. Changes to either of them should propagate to the other one immediately. It demonstrates how to provide and retrieve the value of the math-input in a react-environment.
```
Comment (Nifdee) [inline examples/example1/README.md]:
```suggestion
This simple react app uses both a `MathFieldComponent` and a standard html input control to display the same content. Changes to either of them should propagate to the other one immediately. It demonstrates how to provide and retrieve the value of the math input in a react environment.
```

File: examples/example1/public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  "icons": [
+    {
+      "src": "favicon.ico",
+      "sizes": "64x64 32x32 24x24 16x16",
+      "type": "image/x-icon"
+    }
+  ],
+  "start_url": ".",
+  "display": "standalone",
+  "theme_color": "#000000",
```
Comment (Nifdee) [inline examples/example1/public/manifest.json]: What are these used for?

File: examples/example1/src/index.css [hunk 3]
```diff
@@ -0,0 +1,13 @@
+body {
```
Comment (Nifdee) [inline examples/example1/src/index.css]: I don't think you really need any of this.

File: examples/example1/public/manifest.json (hunk 2 above)
Comment (Khazuar) [reply examples/example1/public/manifest.json]: Removed.

File: examples/example1/src/index.css (hunk 3 above)
Comment (Khazuar) [reply examples/example1/src/index.css]: Removed.

**Pull request merged**